doc/api/tty.md

```diff
@@ -130,3 +131,8 @@ added: v0.5.8
 The `tty.isatty()` method returns `true` if the given `fd` is associated with
 a TTY and `false` if it is not, including whenever `fd` is not a non-negative
 integer.
+
+[`net.Socket`]: net.html#net_class_net_socket
+[`process.stdin`]: process.html#process_process_stdin
+[`process.stdout`]: process.html#process_process_stdout
+[`process.stderr`]: process.html#process_process_stderr
```

Comment:
A nit: out of ABC order.

Reply:
Have we actually been enforcing this? I'm kinda clueless - if we have been generally I'll change it, if not... maybe we should do that separately with linting?

Reply:
Sorry, I thought it was enforced in doc linting or style guide, but it seems I am wrong, I cannot even find PRs that sort references explicitly (I recall there were some). So maybe it is OK as is,

Reply:
For posterity: these are order (I think) in their order of references